filters-share/specific_ELEMHIDE.txt

```diff
@@ -727,7 +727,7 @@ auth.danawa.com###loginBottomBanner
 prod.danawa.com###ebayPowerClickBottomArea
 kimpga.com##div.max-w-screen-lg > .relative > div > div[class=""]:nth-child(1)
 m.danawa.com##.goods-list__ad
-ombudsmannews.com,kpja.kr,hungryapp.co.kr,usjournal.kr,socialvalue.kr,happycampus.com##div[id^="banner"]
+ombudsmannews.com,kpja.kr,hungryapp.co.kr,usjournal.kr,socialvalue.kr,happycampus.com,etorrentro.net,chimhaha.net##div[id^="banner"]
```

Comment:
`etorrentro.net` is an illegal torrent-sharing website.
Therefore, the website has to be isolated from normal websites.